### DISCUSSION TITLE
Search/sort ability in pokemon collection window

### DISCUSSION THREAD

**Discussion by ADEKBC** [Unlucky-Life/ankimon#50] [Ideas]
Please create a search or sort measure for the collection menu. When I encounter a Pokemon, I want to verify if it's one I've caught before, but it is a hassle having to scroll through the entire list just to see if this one Pokemon has been encountered/caught.

**Comment by Unlucky-Life** [comment]
Understandable => Right now the idea is to add a pokeball to the reviewer if this pokemon is in your collection:
![image](https://github.com/Unlucky-Life/ankimon/assets/77027147/ef633281-465a-4640-8e1e-b7eb4c5dae02)

**Reply by ADEKBC** [reply]
That seems like a great way to do it. Looking forward to this update.
